zk-states/src/types.ts

```diff
@@ -128,8 +128,20 @@ interface ProofSuccessUpdate {
   updateType: "proofSuccess";
   callId: string;
 }
+export type initializationProgress =
+  | "initializing"
+  | "compiling program"
+  | "compiling contract"
+  | "creating initial proof"
+  | "initialization done";
```

Comment:
```suggestion
  | "compilingProgram"
  | "compilingContract"
  | "initProof"
  | "done";
```

these are not strings to be rendered, but cases, so name them as variables